### DISCUSSION TITLE
Internal Bevy XR Crates

### DISCUSSION THREAD

**Discussion by mrchantey** [bevyengine/bevy#8896] [Ideas]
Now that [`RenderTarget::TextureView`](https://github.com/bevyengine/bevy/pull/8042) has been merged we may start to see external XR crates appearing. It might be worth getting ahead of crate organization sooner rather than later.
## Current Status
- `bevy_xr` reserved by bevy
- `bevy_openxr` currently owned by @blaind
- `bevy_webxr` currently available
- `bevy_vision_os` currently available

It may be possible to cram the implementations into `bevy_xr` but to me that sounds messy. I'm guessing `bevy_xr` would contain the common traits etc, and the others would contain implementations and device specific features. If we intend to have internal XR implementation crates should we start transferring and reserving the ones we'd need?

**Comment by IceSentry** [comment]
Either way, I think it may be worth it to add those available ones to https://github.com/bevyengine/bevy-crate-reservations
